multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/monitoring/FlowableJobExecutorInformation.java

```diff
@@ -37,8 +37,8 @@ private boolean isPastCacheTimeout() {
     }
 
     private void updateCurrentJobExecutorQueueSize() {
-        currentJobExecutorQueueSize = jobExecutor.getThreadPoolQueue()
-                                                 .size();
+        currentJobExecutorQueueSize = processEngineConfiguration.getAsyncExecutorThreadPoolQueue()
```

Comment:
This metric should currently shechuled jobs waiting in the queue of job/task executor. Is this working in the same way?